cli/util/util_test.go

```diff
@@ -169,11 +170,19 @@ func TestAwaitUtilCancelTx(t *testing.T) {
 
 	e.In.WriteString("one\r")
 	e.Run(t, append(args, txHash.StringLE())...)
-	e.CheckNextLine(t, "Conflicting transaction accepted")
-	resHash, _ := e.CheckAwaitableTxPersisted(t)
 
-	require.Eventually(t, func() bool {
-		_, aerErr := e.Chain.GetAppExecResults(resHash.Hash(), trigger.Application)
-		return aerErr == nil
-	}, time.Second*2, time.Millisecond*50)
+	response := e.GetNextLine(t)
+	if strings.Contains(response, "Conflicting transaction accepted") {
+		resHash, _ := e.CheckAwaitableTxPersisted(t)
+		require.Eventually(t, func() bool {
+			_, aerErr := e.Chain.GetAppExecResults(resHash.Hash(), trigger.Application)
+			return aerErr == nil
+		}, time.Second*2, time.Millisecond*50)
+	} else {
```

Comment:
Else should check that `response` contains expected string about target transaction acceptance. And if not, then test should fail.